ext/src/ruby_api/store.rs

```diff
@@ -163,40 +163,28 @@ impl Store {
     }
 
     /// @yard
-    /// Returns the amount of fuel consumed by this {Store}’s execution so far,
-    /// or +nil+ when the {Engine}’s config does not have fuel enabled.
-    /// @return [Integer, Nil]
-    pub fn fuel_consumed(&self) -> Option<u64> {
-        self.inner_ref().fuel_consumed()
+    /// Returns the amount of fuel in the {Store}.
+    /// This function errors if fuel consumption is not configured by passing
+    /// enabling fuel consumption via Wasmtime::Engine::new.
+    ///
+    /// @return [Integer, Error]
+    pub fn get_fuel(&self) -> Result<u64, Error> {
+        self.inner_ref().get_fuel().map_err(|e| error!("{}", e))
     }
 
     /// @yard
-    /// Adds fuel to the {Store}.
+    /// Sets fuel to the {Store}.
     /// @param fuel [Integer] The fuel to add.
-    /// @def add_fuel(fuel)
-    /// @return [Nil]
-    pub fn add_fuel(&self, fuel: u64) -> Result<(), Error> {
+    /// @def set_fuel(fuel)
+    /// @return [Error]
```

Comment:
From Ruby, this won't _return_ an error but raise it. So the return value would always be nil.

Reply:
Same as my comment above, I've added `@raise`

Reply:
Oh, that's nice 👍. We don't apply that consistently in the library but we should!